README.md

```diff
@@ -1,205 +1,92 @@
-  <a href="https://github.com/meilisearch/meilisearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
   <a href="https://app.bors.tech/repositories/26457"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine 🔍</p>
+<p align="center">⚡ An open-source, lightning-fast, and hyper-relevant search engine that fits effortlessly into your apps, websites, and workflow 🔍</p>
 
-**Meilisearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
-For more information about features go to [our documentation](https://docs.meilisearch.com/).
+Meilisearch's mission is to help you shape a delightful search experience in a snap. To fulfill it, Meilisearch offers out-of-the-box features to speed up your workflow.
 
-<p align="center">
-  <img src="assets/trumen-fast.gif" alt="Web interface gif" />
+<p align="center" name="demo">
+  <img src="assets/demo-light.gif#gh-light-mode-only" alt="demo-light">
+  <img src="assets/demo-dark.gif#gh-dark-mode-only" alt="demo-light">
 </p>
 
-## ✨ Features
-* Search-as-you-type experience (answers < 50 milliseconds)
-* Full-text search
-* Typo tolerant (understands typos and misspelling)
-* Faceted search and filters
-* Supports hanzi (Chinese characters)
-* Supports synonyms
-* Easy to install, deploy, and maintain
-* Whole documents are returned
-* Highly customizable
-* RESTful API
-
-## Getting started
-
-### Deploy the Server
-
-#### Homebrew (Mac OS)
-
-```bash
-brew update && brew install meilisearch
-meilisearch
-```
-
-#### Docker
-
-```bash
-docker run -p 7700:7700 -v "$(pwd)/meili_data:/meili_data" getmeili/meilisearch
-```
-
-#### Announcing a cloud-hosted Meilisearch
-
-Join the closed beta by filling out this [form](https://meilisearch.typeform.com/to/FtnzvZfh).
-
-#### Try Meilisearch in our Sandbox
-
-Create a Meilisearch instance in [Meilisearch Sandbox](https://sandbox.meilisearch.com/). This instance is free, and will be active for 48 hours.
-
-#### Run on Digital Ocean
-
-[![DigitalOcean Marketplace](assets/do-btn-blue.svg)](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101)
-
-#### Deploy on Platform.sh
-
-<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/meilisearch/.platform.template.yaml&utm_content=meilisearch&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
-</a>
+<!-- Go to [this demo]() to try Meilisearch! -->
```

Comment:
I'd say the best-looking one it's the new movie demo "Where to watch", it would be great if a sort-by and/ or  a faceted search could be added eventually

Reply:
For me, it should be the movies demo as highlighted by the gifs.

Reply:
I discussed with Morgane D. who discussed with Quentin: they want to wait for v0.28.0 to be released before publishing the movie demo officially. Depending on when we will merge this PR and the time it takes to do the demo update (Morgane also other priorities), I will put or not the current link or wait for the demo update to be done.
In any case, Morgane D. is aware of our need and she's doing her best to provide the demo 🥰

Reply:
@mdubus Were we able to make progress on the demo (Asking that since v0.28 has been released 😎)? Thanks!